packages/vuetify/src/components/VDataTable/__tests__/VDataTable.spec.ts

```diff
@@ -348,6 +348,24 @@ describe('VDataTable.ts', () => {
     expect(fn).toHaveBeenCalled()
   })
 
+  it('should emit event when double-clicking on internally created row', async () => {
```

Comment:
that's the same test as for `click:row` and (incoming) `contextmenu:row`, we could make it dry by iterating over `['click', 'dblclick', 'contextmenu']` array

Reply:
Good suggestion. I was thinking that too.

Reply:
I'll do that with whichever PR goes in last.

Reply:
https://jestjs.io/docs/en/api#describeeachtablename-fn-timeout